### ISSUE TITLE
Spaces in job name causes bug in queue/until_front_of_line

### ISSUE THREAD

**Issue by spirosekulovski** [eddiewebb/circleci-queue#128]
### Orb version
3.1.4

### What happened
 The command **queue/until_front_of_line** does not work when the name of the job contains spaces. This may also affect other jobs/commands. 

This is the output of a CircleCI Job when job name contains spaces. Job is running alone, no other concurrent job at the moment of grabbing this log:
```
Block: false
Expecting CCI Personal Access TOKEN Named: CIRCLECI_API_KEY
Using API key for user: "<-redacted->" on host https://circleci.com
staging matches queueable branch names
Max Queue Time: 600 seconds.
Only blocking execution if running previous jobs on branch: staging
Getting queue ordering
Workflow: <-redacted-> is from pipeline #169
Orb parameter block-workflow is false. Use Job level queueing.
Only blocking execution if running previous jobs matching this job: release job name with spaces
API Call for existing jobs returned no matches. This means job is alone.
This Job's Pipeline #: 169
Front of Queue (fifo) Pipeline #: 
This build (307), pipeline (169) is queued, waiting for build(null) pipeline () to complete.
Total Queue time: 0 seconds.
Only blocking execution if running previous jobs on branch: staging
Getting queue ordering
Workflow: <-redacted-> is from pipeline #169
Orb parameter block-workflow is false. Use Job level queueing.
Only blocking execution if running previous jobs matching this job: release job name with spaces
API Call for existing jobs returned no matches. This means job is alone.
This Job's Pipeline #: 169
Front of Queue (fifo) Pipeline #: 
This build (307), pipeline (169) is queued, waiting for build(null) pipeline () to complete.
Total Queue time: 11 seconds.
Only blocking execution if running previous jobs on branch: staging
Getting queue ordering
Workflow: <-redacted-> is from pipeline #169
Orb parameter block-workflow is false. Use Job level queueing.
Only blocking execution if running previous jobs matching this job: release job name with spaces
API Call for existing jobs returned no matches. This means job is alone.
This Job's Pipeline #: 169
Front of Queue (fifo) Pipeline #: 
This build (307), pipeline (169) is queued, waiting for build(null) pipeline () to complete.
Total Queue time: 22 seconds.
Only blocking execution if running previous jobs on branch: staging
Getting queue ordering
Workflow: <-redacted-> is from pipeline #169
Orb parameter block-workflow is false. Use Job level queueing.
Only blocking execution if running previous jobs matching this job: release job name with spaces
API Call for existing jobs returned no matches. This means job is alone.
This Job's Pipeline #: 169
Front of Queue (fifo) Pipeline #: 
This build (307), pipeline (169) is queued, waiting for build(null) pipeline () to complete.
Total Queue time: 33 seconds.
```
It seems as if the command that checks for other concurrent jobs fails because the name of the job has spaces.

This is the output of a CircleCI Job when the job name contains underscores instead of spaces:
```
Block: false
Expecting CCI Personal Access TOKEN Named: CIRCLECI_API_KEY
Using API key for user: "<-redacted->" on host https://circleci.com
staging matches queueable branch names
Max Queue Time: 600 seconds.
Only blocking execution if running previous jobs on branch: staging
Getting queue ordering
Workflow: <-redacted-> is from pipeline #168
Orb parameter block-workflow is false. Use Job level queueing.
Only blocking execution if running previous jobs matching this job: release_job_name_without_spaces
This Job's Pipeline #: 168
Front of Queue (fifo) Pipeline #: 168
API shows no conflicting jobs/workflows. However it is possible a previous workflow has pending jobs not yet visible in API. To avoid a race condition we will verify out place in queue.
Rerunning check 1/1
Only blocking execution if running previous jobs on branch: staging
Getting queue ordering
Workflow: <-redacted-> is from pipeline #168
Orb parameter block-workflow is false. Use Job level queueing.
Only blocking execution if running previous jobs matching this job: release_job_name_without_spaces
This Job's Pipeline #: 168
Front of Queue (fifo) Pipeline #: 168
Front of the line, WooHoo!, Build continuing
```
### Expected behavior
Job name should have no influence over the queue/until_front_of_line or other commands/jobs.

**Comment by eddiewebb** [comment]
FIXED with major thanks to @rainofterra going above and beyond on that fix.

**Comment by eddiewebb** [comment]
https://circleci.com/developer/orbs/orb/eddiewebb/queue `3.1.5`

**Comment by RainofTerra** [comment]
@eddiewebb not _quite_ fixed as it turns out, though #134 should do it (and there is now a #135 too)

I finally decided to properly setup a dev namespace for myself to publish dev orbs on so this should actually work now, I tested it in our pipelines.

**Comment by eddiewebb** [comment]
`3.1.6` includes jq fix for spaces in job name.

(`3.2.0` includes 429 support fro #135 )
